fe1-web/src/core/objects/__tests__/Timestamp.test.ts

```diff
@@ -1,15 +1,14 @@
 import 'jest-extended';
+import { FIXED_SYSTEM_TIME } from '__tests__/utils';
 
 import { Timestamp } from '../Timestamp';
 
+jest.useFakeTimers('modern');
+jest.setSystemTime(FIXED_SYSTEM_TIME);
```

Comment:
I think that you need to add `jest.useRealTimers()` at the end of this file to ensure that it doesn't change other tests. It should have been done since the beginning but it probably have been forgotten by mistake.

Reply:
Oh I see! Thank you :)